CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Fix issue with `buf lint` where comment ignores in the shape of `// buf:lint:ignore <RULE_ID> <extra comment>`
+  are not recognized due to the extra comment.
```

Comment:
`were not recognized` not `are not recognized`

Reply:
fixed in [78c78e6](https://github.com/bufbuild/buf/pull/3295/commits/78c78e6e49151ff5d133f49ab14533deae75336b)